examples/blog/config.js

```diff
@@ -1,438 +1,434 @@
-                    .template('<send-email post="entry"></send-email>')
-            ]);
-
-
-        comment.listView()
-            .title('Comments')
-            .perPage(10) // limit the number of elements displayed per page. Default is 30.
-            .fields([
-                nga.field('created_at', 'date')
-                    .label('Posted'),
-                nga.field('author'),
-                nga.field('body', 'wysiwyg')
-                    .stripTags(true)
-                    .map(truncate),
-                nga.field('post_id', 'reference')
-                    .label('Post')
-                    .map(truncate)
-                    .targetEntity(post)
-                    .targetField(nga.field('title').map(truncate))
-            ])
-            .filters([
-                nga.field('q', 'template')
-                    .label('')
-                    .pinned(true)
-                    .template('<div class="input-group"><input type="text" ng-model="value" placeholder="Search" class="form-control"></input><span class="input-group-addon"><i class="glyphicon glyphicon-search"></i></span></div>'),
-                nga.field('created_at', 'date')
-                    .label('Posted')
-                    .attributes({'placeholder': 'Filter by date'}),
-                nga.field('post_id', 'reference')
-                    .label('Post')
-                    .targetEntity(post)
-                    .targetField(nga.field('title'))
-                    .remoteComplete(true, { refreshDelay: 300 })
-            ])
-            .listActions(['edit', 'delete']);
-
-        comment.creationView()
-            .fields([
-                nga.field('created_at', 'date')
-                    .label('Posted')
-                    .defaultValue(new Date()), // preset fields in creation view with defaultValue
-                nga.field('author'),
-                nga.field('body', 'wysiwyg'),
-                nga.field('post_id', 'reference')
-                    .label('Post')
-                    .map(truncate)
-                    .filters(function(search) {
-                        if (!search) {
-                            return;
-                        }
-
-                        return {
-                            q: search // Full-text search
-                        };
-                    })
-                    .targetEntity(post)
-                    .targetField(nga.field('title'))
-                    .validation({ required: true })
-                    .remoteComplete(true, { refreshDelay: 0 })
-            ]);
+        }
+        return { params: params };
+    });
+}]);
 
-        comment.editionView()
-            .fields(comment.creationView().fields())
-            .fields([nga.field(null, 'template')
-                .label('')
-                .template('<post-link entry="entry"></post-link>') // template() can take a function or a string
-            ]);
+var Factory = require('admin-config/lib/Factory');
```

Comment:
We can't make webpack a requirement at that stage. It's nice to support it, but the "Getting started" tutorial must be as simple as possible. Therefore, the definition of the DIC value must be in a `config()` or a `run()`, and get the factory from DIC.